packages/xo-web/src/common/xo/index.js

```diff
@@ -1141,7 +1141,8 @@ export const deleteTemplates = templates =>
     await Promise.all(
       map(resolveIds(templates), id =>
         _call('vm.delete', { id }).catch(reason => {
-          if (vmIsTemplate.is(reason)) {
+          const template = getObject(store.getState(), id)
+          if (forbiddenOperation.is(reason) && template.isDefaultTemplate) {
```

Comment:
So the error here might as well be that `destroy` is a blocked operations, right?

Reply:
@pdonias Is this comment resolved?

Reply:
Yes.